app/code/Magento/Catalog/view/adminhtml/templates/catalog/product/tab/alert.phtml

```diff
@@ -4,6 +4,9 @@
  * See COPYING.txt for license details.
  */
 
+/**
+ * @var \Magento\Framework\Escaper $escaper
+ */
 ?>
```

Comment:
Please remove not needed opening and closing PHP tags

Reply:
fixed